ui/login-callback/index.jsx

```diff
@@ -3,8 +3,8 @@ import { render } from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
 import 'font-awesome/css/font-awesome.css';
 
-import LoginCallback from './js/auth/LoginCallback';
-import './css/login.css';
+import LoginCallback from './LoginCallback';
+import '../css/login.css';
```

Comment:
I wonder if eventually in the future, we should move the app/component-specific CSS alongside the apps/components themselves? Perhaps also worth seeing if we should use one of the CSS-in-JS approaches that are becoming popular now.

Reply:
Yeah, I like this idea.  CSS-in-JS seems pretty cool when there's a very specific tweak we need.  And I really like the idea of getting the rest of the CSS closer to the components they affect.  :)